blosc/blosc2.h

```diff
@@ -711,6 +731,24 @@ BLOSC_EXPORT void blosc_cbuffer_versions(const void* cbuffer, int* version,
  */
 BLOSC_EXPORT const char* blosc_cbuffer_complib(const void* cbuffer);
 
+/*********************************************************************
+  Structures and functions related with user-defined input/output.
+*********************************************************************/
+typedef struct {
+  void* (*open)(const char *urlpath, const char *mode, void *params);
```

Comment:
Consider using typedefs for these callbacks.
```
typedef void* (*blosc2_open_cb)(const char *urlpath, const char *mode, void *params);
typedef int (*blosc2_close_cb)(void *stream, void* params);
```
Then in your struct you just have:
```
typedef struct {
  blosc2_open_cb open;
  blosc2_close_cb close;
  ...
} blosc2_io;
```

Reply:
If you look in my minizip-ng project I have done something similar:
https://github.com/zlib-ng/minizip-ng/blob/master/mz_strm.h

Reply:
I agree, thank you very much for the suggestion!